spec/feature/account_home_spec.rb

```diff
@@ -0,0 +1,22 @@
+require("rails_helper")
+
+RSpec.describe("AccountHome", type: :feature) do
+  before do
+    stub_content_store_has_item("/")
+  end
+
+  context("/account/home") do
+    it("redirect users to One Login's Your Services page") do
```

Comment:
I'm guessing the script just converted `should` to `it` to get the test expectations working and that the test names will need to be manually modified by a human later?

Reply:
Yes, in fact I've done that now as part of a general clean-up (I'll add details of my arbitrary style decisions in the description so they can be critiqued).